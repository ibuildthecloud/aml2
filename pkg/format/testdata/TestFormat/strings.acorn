a: """multiline"""
ab: """
multiline
"""
ac: """
  multiline
"""
ad: `
  multiline

  `
