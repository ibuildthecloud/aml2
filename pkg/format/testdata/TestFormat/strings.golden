&errors.errorString{s: "parse: string literal not terminated (and 1 more errors)"}
